pkg/csr/csr.go

```diff
@@ -325,11 +325,7 @@ func findHostName(instanceInfo *instance.Info, instanceSigner ssh.Signer) (strin
 		return "", fmt.Errorf("error instantiating Windows instance: %w", err)
 	}
 	// get the instance host name  by running hostname command on remote VM
-	hostName, err := win.Run("hostname", true)
-	if err != nil {
-		return "", fmt.Errorf("error getting the host name, with stdout %s: %w", hostName, err)
-	}
-	return hostName, nil
+	return win.GetHostname()
```

Comment:
What does this return normally, when not part of AD?
Is it the same as before, just the hostname?

Reply:
And have you been able to see what the csr approval behavior is like when using AD?

Reply:
> What does this return normally, when not part of AD? Is it the same as before, just the hostname?

correct, the proposed implementation keeps backward compatibility  the previous. that is, if no DNS suffix found just the hostname is returned.

Reply:
> And have you been able to see what the csr approval behavior is like when using AD?

Same, I had to change this here, given that the approver could not match a node by name using just the output of  `hostname,` it also helps to standardize the "get hostname" implementation to be consistent across the code base.